src/components/CreateEventAttestationButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { useCreateEventAttestation } from "../hooks/useCreateEventAttestation";
+import type {Event} from "../hooks/useCreateEventAttestation"
+
+export function CreateEventAttestationButton() {
+    const createEventAttestation = useCreateEventAttestation();
+
+    async function handleCreateEventAttestation() {
+
+        const today = new Date();
+        const tomorrow = new Date();
+        tomorrow.setDate(today.getDate()+1);
+        const newEvent = {
+            owner:"0xa90914762709441d557De208bAcE1edB1A3968b2",
+            name:"test",
+            briefDescription:"test description",
+            fullDescription:"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate, numquam! Esse officia molestiae illum nihil dolorum rem debitis laborum ab. Quaerat voluptatibus repellendus deserunt porro distinctio laboriosam earum nesciunt laborum?",
+            startsAt: today.valueOf(),
+            endsAt: tomorrow.valueOf(),
+            imageUrl:"https://picsum.photos/id/237/200/300",
+        } as Event
```

Comment:
I understand you're hardcoding this -- `#would` put it in a variable outside the component as a constant (I know this is JS but I think PEP8 has a good point about constants: https://peps.python.org/pep-0008/#constants